app.py

```diff
@@ -1,9 +1,91 @@
-from flask import Flask, render_template
+from flask import Flask, render_template, request, flash, redirect, url_for
+from database.db import Database
+from flask_login import LoginManager, login_user, logout_user, login_required, UserMixin
+from config import Config
+from forms import SignUpForm, LoginForm
+
 app = Flask(__name__)
+app.config.from_object(Config)
+
+db = Database(app, "cs361_alberjes", 3526)
```

Comment:
👀 is this your password? Do you want to take that out so that other people aren't using it?

Reply:
Do you recommend me connecting it to my own DB to test?

Reply:
Yes, just replace with your ONID and last 4 of your student number